staging/src/k8s.io/apiserver/pkg/storage/errors.go

```diff
@@ -46,6 +48,7 @@ var errCodeToMessage = map[int]string{
 	ErrCodeInvalidObj:               "invalid object",
 	ErrCodeUnreachable:              "server unreachable",
 	ErrCodeTimeout:                  "request timeout",
+	ErrCodeCorruptedData:            "corrupted data",
```

Comment:
why not undecryptable data? is this covering another scenario aswell?

Reply:
As requested in the KEP, we're also handling corrupted data with this error.